climada/entity/exposures/base.py

```diff
@@ -906,15 +906,29 @@ def copy(self, deep=True):
             **metadata
         )
 
-    def write_raster(self, file_name, value_name='value', scheduler=None):
+    def write_raster(self, file_name, value_name=['value'], scheduler=None):
```

Comment:
```suggestion
    def write_raster(self, file_name, value_name='value', scheduler=None):
```

As agreed in the Developers' Guidelines, we should not use mutables (here a list) as defaults.